mypy/util.py

```diff
@@ -134,3 +140,36 @@ def id(self, o: object) -> int:
             self.id_map[o] = self.next_id
             self.next_id += 1
         return self.id_map[o]
+
+
+if sys.version_info >= (3, 6):
+    PathType = Union[AnyStr, os.PathLike]
```

Comment:
@ilevkivskyi  it works! I thought strings can only help with forward declarations, but I guess they just tell runtime not to worry about it whatever the cause might be.